src/test/scala/com/amazon/deequ/suggestions/ConstraintSuggestionsIntegrationTest.scala

```diff
@@ -47,10 +47,10 @@ class ConstraintSuggestionsIntegrationTest extends WordSpec with SparkContextSpe
       val categories = Array("DE", "NA", "IN", "EU")
 
       val records = (0 until numRecords)
-        .map { _ =>
+        .map { record =>
```

Comment:
this is not required as the chance for a collision in UUID is really low

Reply:
I found that on some occasions (needed 6 tries just now to reproduce this) the line where we assert that we suggest an isUnique constraint fails. The explanation I could imagine for that is that the randomness combined with the way we calculate the approximate Uniqueness using the HLL sketch could be a problem. So there aren't actual collisions but the approximate Uniqueness is not accurate enough and too low. Is that possible?